src/group-configurations/index.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import {
+  Container, Layout, Stack, Row,
+} from '@edx/paragon';
+
+import { LoadingSpinner } from '../generic/Loading';
+import { useModel } from '../generic/model-store';
+import SubHeader from '../generic/sub-header/SubHeader';
+import getPageHeadTitle from '../generic/utils';
+import messages from './messages';
+import ContentGroupsSection from './content-groups-section';
+import ExperimentConfigurationsSection from './experiment-configurations-section';
+import EnrollmentTrackGroupsSection from './enrollment-track-groups-section';
+import { useGroupConfigurations } from './hooks';
+
+const GroupConfigurations = ({ courseId }) => {
+  const { formatMessage } = useIntl();
+  const courseDetails = useModel('courseDetails', courseId);
+  const {
+    isLoading,
+    groupConfigurations: {
+      allGroupConfigurations,
+      shouldShowEnrollmentTrack,
+      shouldShowExperimentGroups,
+      experimentGroupConfigurations,
+    },
+  } = useGroupConfigurations(courseId);
+
+  document.title = getPageHeadTitle(
+    courseDetails?.name,
+    formatMessage(messages.headingTitle),
+  );
+
+  if (isLoading) {
+    // eslint-disable-next-line react/jsx-no-useless-fragment
```

Comment:
I guess `eslint-disable..` can be removed

Reply:
sure, removed